ui/apps/platform/cypress/integration/violations/Violations.selectors.ts

```diff
@@ -1,13 +1,15 @@
 export const selectors = {
     actions: {
         btn: 'td .pf-v6-c-menu-toggle[aria-label="Kebab toggle"]', // via ActionsColumn element
-        excludeDeploymentBtn: 'button:contains("Exclude deployment")',
-        resolveBtn: 'button:contains("Mark as resolved")',
-        resolveAndAddToBaselineBtn: 'button:contains("Resolve and add to process baseline")',
+        excludeDeploymentBtn:
+            '.pf-v6-c-menu__list button:contains("Exclude deployment from policy")',
+        resolveBtn: '.pf-v6-c-menu__list button:contains("Mark as resolved")',
+        resolveAndAddToBaselineBtn:
+            '.pf-v6-c-menu__list button:contains("Resolve and add to process baseline")',
     },
     details: {
-        title: 'h1.pf-v6-c-title',
-        subtitle: 'section:has(> h1) > p',
+        title: '[data-ouia-component-id="PageHeader-title"]',
+        subtitle: '[data-ouia-component-id="PageHeader-subtitle"]',
```

Comment:
Could we move these to the pf6.ts selectors file? These are good candidates for a general library of selectors that we can use with PatternFly for more reliable selectors (and hopefully smoother future migrations).

Reply:
Makes sense. Here's another commit for that https://github.com/stackrox/stackrox/pull/18194/changes/f01fc0a9859646e0a6dc22f6f0aeccb122f6e3b1